app/controllers/concerns/europeana/catalog.rb

```diff
@@ -83,6 +83,12 @@ def previous_and_next_document_params(index, window = 1)
       api_params
     end
 
+    def fetch_with_hierarchy(id = nil, extra_controller_params = {})
```

Comment:
Unused method argument - `extra_controller_params`. If it's necessary, use `_` or `_extra_controller_params` as an argument name to indicate that it won't be used.